pmm-app/src/pmm-qan/panel/qan.constants.ts

```diff
@@ -314,7 +311,7 @@ export const METRIC_CATALOGUE = {
   },
   filesort: {
     humanizeName: 'Filesort',
-    tooltipText: 'The query used a filesort',
+    tooltipText: 'The query used a file sort',
```

Comment:
filesort  is a common term in MySQL  (https://www.percona.com/blog/2009/03/05/what-does-using-filesort-mean-in-mysql/) so probably, it's better to keep it